### PR TITLE
docs: Update broken links

### PR DESCRIPTION
**Pull request by Timtech4u**
This is to update broken links I've found on the Garden's documentation to their correct links.

A few updates also needs to be done to the Garden's [website](https://garden.io)

e.g. `Stack Graph` references on the website should be updated. cc: @edvald

### CODE REVIEW

File: docs/reference/providers/kubernetes.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Kubernetes clusters, and adds the [`helm`](https://docs.garden.io/reference/modu
 
 For usage information, please refer to the [guides section]https://docs.garden.io/guides). A good place to start is
 the [Remote Kubernetes guide](https://docs.garden.io/guides/remote-kubernetes) guide if you're connecting to remote clusters.
-The [demo-project](https://docs.garden.io/examples/demo-project) example project and guide are also helpful as an introduction.
+The [demo-project](https://docs.garden.io/example-projects/demo-project) example project and guide are also helpful as an introduction.
```
Comment (edvald) [inline docs/reference/providers/kubernetes.md]: This file is auto-generated, to the link needs to be fixed here: https://github.com/garden-io/garden/blob/1ffcb98713054731cef284bf81423d442c095c21/garden-service/src/plugins/kubernetes/local/local.ts#L23

File: docs/reference/providers/local-kubernetes.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ tocTitle: "`local-kubernetes`"
 
 The `local-kubernetes` provider is a specialized version of the [`kubernetes` provider](https://docs.garden.io/reference/providers/kubernetes) that automates and simplifies working with local Kubernetes clusters.
 
-For general Kubernetes usage information, please refer to the [guides section](https://docs.garden.io/guides). For local clusters a good place to start is the [Local Kubernetes guide](https://docs.garden.io/guides/local-kubernetes) guide. The [demo-project](https://docs.garden.io/examples/demo-project) example project and guide are also helpful as an introduction.
+For general Kubernetes usage information, please refer to the [guides section](https://docs.garden.io/guides). For local clusters a good place to start is the [Local Kubernetes guide](https://docs.garden.io/guides/local-kubernetes) guide. The [demo-project](https://docs.garden.io/example-projects/demo-project) example project and guide are also helpful as an introduction.
```
Comment (edvald) [inline docs/reference/providers/local-kubernetes.md]: This file is auto-generated, to the link needs to be fixed here: https://github.com/garden-io/garden/blob/7ed4648eac131453d0465f6822f02397ef788361/garden-service/src/plugins/kubernetes/kubernetes.ts#L186

File: docs/reference/providers/kubernetes.md (hunk 1 above)
Comment (edvald) [reply docs/reference/providers/kubernetes.md]: After that is fixed, running `npm run generate-docs` will render the markdown file.

**Comment by Timtech4u** [comment]
Pushed some new commits @edvald

**Comment by edvald** [comment]
Looks alright, but still looks like you need run `npm run generate-docs` again and commit the changes, so the linting check will pass.

**Comment by Timtech4u** [comment]
> Looks alright, but still looks like you need run `npm run generate-docs` again and commit the changes, so the linting check will pass.

Faced some errors doing that, I've ran `npm i` to ensure the modules are installed.

```
Error: Cannot find module '/home/tim/Codes/garden/garden-service/build/src/bin/generate-docs.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
Screenshot from 2020-03-17 08-58-02

**Comment by eysi09** [comment]
Hi @Timtech4u!

Did you follow the steps in the CONTRIBUTING.md guide: https://github.com/garden-io/garden/blob/master/CONTRIBUTING.md#step-4-bootstrap-project?

You need to follow the steps there to bootstrap the project and install NPM dependencies. 

You might also need some of the dev dependencies from this script: https://github.com/garden-io/garden/blob/master/bin/bootstrap-osx, in particular `parallel`.  You should be able to install it with package managers on macOS and Linux.

**Comment by edvald** [comment]
Hey @Timtech4u. I ended up merging your changes with #1701, so I'll close this one. Thanks for all these fixes, we sure bungled all this when we refactored our docs 😬

**Pull request closed without merging**